pkgs/applications/networking/browsers/chromium/update.nix

```diff
@@ -64,7 +64,7 @@ in rec {
     csv2nix = name: src: import (runCommand "${name}.nix" {
       src = builtins.fetchurl src;
     } ''
-      esc() { echo "\"$(echo "$1" | sed -e 's/"\\$/\\&/')\""; }
+      esc() { echo "\"$(echo "$1" | sed -e 's/"\\$/\\&/')\""; } # ohai emacs "
```

Comment:
Nah, it’s the emacs syntax recognition that is buggy here. To be fair, it’s not trivial to parse nix’s arbitrarily nested splices. And in Emacs syntax hilighting is mostly done with regexes afaik.